specification/repository.md

```diff
@@ -91,9 +91,7 @@ approval is granted, GDI projects MUST NOT cut a GA release.
 - MUST add [CODE_OF_CONDUCT.md](templates/CODE_OF_CONDUCT.md)
 - MUST add [CONTRIBUTING.md](templates/CONTRIBUTING.md)
 - MUST have a [.github/CODEOWNERS](templates/.github/CODEOWNERS) file with a maintainers team
-- SHOULD have a
-  [.github/workflows/request-reviews.yml](templates/.github/workflows/request-reviews.yml)
-  to automate requesting reviews from approvers
+- SHOULD have an approvers team in CODEOWNERS
```

Comment:
How does this work together with the previous line?

Reply:
works fine I think -- maintainers required, approvers optional. Top of this doc also states the same thing